api/src/main/java/org/openmrs/Order.java

```diff
@@ -363,6 +363,38 @@ public void setCommentToFulfiller(String commentToFulfiller) {
 	}
 	
 	/**
+	 * Convenience method to determine if the order is activated as of the current date
+	 * 
+	 * @return boolean indicating whether the order was activated before or on the current date
+	 * @since 2.0
+	 * @see #isActivated(java.util.Date)
```

Comment:
@wluyima as I already told you isActive(), isActive(null), isStarted() and isStarted(null) don't have the test you are asking me to write. Tell me how to test this and I will.